src/cfe.c

```diff
@@ -68,8 +68,12 @@ extern void cfe(
     double primary_flux=0.0;      // pointers to these variables passed to conceptual nonlinear reservoir which has two outlets, primary & secondary
     double secondary_flux=0.0;    // pointers to these variables passed to conceptual nonlinear reservoir which has two outlets, primary & secondary
     double lateral_flux=0.0;      // flux from soil to lateral flow Nash cascade +to cascade  [m/timestep]
-    double percolation_flux=0.0;  // flux from soil to gw nonlinear researvoir, +downward  [m/timestep] 
-    
+
+    double percolation_flux=0.0;  // flux from soil to gw nonlinear researvoir, +downward  [m/timestep]
+
+    // store current soil storage_m in a temp. variable
+    double storage_temp_m = soil_reservoir_struct->storage_m;

```

Comment:
Should this be done after ET?  Probably won't matter since ET won't overlap much with frozen soils...